lib/evaluateFunctions.js

```diff
@@ -65,6 +68,229 @@ function evaluateFunctions(node) {
   return NodeStatus.noChange(node);
 }
 
+// Evaluate nthRoot() function
+// Returns a NodeStatus object.
+function nthRoot(node) {
+  if (!NodeType.isFunction(node, 'nthRoot')) {
+    return NodeStatus.noChange(node);
+  }
+
+  let oldNode = node;
+  let newNode = clone(node, false);
+  const radicandNode = newNode.args[0];
+  const rootNode = newNode.args.length === 2 ? newNode.args[1] : NodeCreator.constant(2);
+
+  if (NodeType.isOperator(radicandNode)) {
+    // if is operator and operation is addition, we can't do anything
+    if (radicandNode.op === '+' || radicandNode.op === '-') {
+      return NodeStatus.noChange(node);
+    }
+    else if (radicandNode.op === '*' || radicandNode.op === '/') {
+      // distribute the root into the children
+      const subSteps = [];
+      const children = [];
+      for (let i = 0; i < radicandNode.args.length; i++) {
+        const child = radicandNode.args[i];
+        children.push(NodeCreator.nthRoot(child, rootNode));
+      }
+      newNode = NodeCreator.operator(radicandNode.op, children);
+      subSteps.push(NodeStatus.nodeChanged(
+        MathChangeTypes.DISTRIBUTE_NTH_ROOT, oldNode, newNode));
+
+      oldNode = NodeStatus.resetChangeGroups(newNode);
+      newNode = clone(oldNode, false);
+
+      for (let i = 0; i < newNode.args.length; i++) {
+        const child = radicandNode.args[i];
+        const childNodeStatus = nthRoot(child);
+        if (childNodeStatus.hasChanged()) {
+          newNode.args[i] = childNodeStatus.newNode;
+          subSteps.push(NodeStatus.childChanged(newNode, childNodeStatus, i));
+        }
+      }
+
+      return NodeStatus.nodeChanged(
+        MathChangeTypes.NTH_ROOT_VALUE, oldNode, newNode, true, subSteps);
+    }
+    else if (radicandNode.op === '^') {
+      const baseNode = radicandNode.args[0];
+      const exponentNode = radicandNode.args[1];
+      // TODO(ael): this only works if they actually have a value...
+      // ideally I'd use equals, but the nodes are different for some reason
+      // even if the value is the same...
+      if (rootNode.value === exponentNode.value) {
+        newNode = baseNode;
+        return NodeStatus.nodeChanged(
+          MathChangeTypes.NTH_ROOT_VALUE, oldNode, newNode);
+      } else {
+        return NodeStatus.noChange(node);
```

Comment:
there are some other cases we'd want to catch to probably

mainly I'm thinking of exponent % rootnode value === 0

e.g. `sqrt(x^4)`